package.json

```diff
@@ -16,7 +16,8 @@
     "next": "13.2.3",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "styled-components": "^5.3.8"
+    "styled-components": "^5.3.8",
+    "test": "^3.3.0"
```

Comment:
I'm not sure about this library? Did you install it instead of using the script `npm run test`?